tests/api/test_api.py

```diff
@@ -154,11 +154,11 @@ async def test_if_get_grundsteuer_job_returns_success_status_with_result():
         mock_service = MagicMock(get_request_by_request_id=MagicMock(return_value=erica_request))
         get_service_mock.return_value = GrundsteuerService(service=mock_service)
         response = await get_grundsteuer_job(request_id)
-        assert response.processStatus == JobState.SUCCESS
+        assert response.process_status == JobState.SUCCESS
         assert response.result.pdf == pdf
         assert response.result.transferticket == transferticket
```

Comment:
Should also be snake_case right --> transfer_ticket (to be changed in the ouput dto too)

Reply:
We changed that in #99 because transfer ticket is just one word.

Reply:
Ah ok I missed that PR. Although it is only 1 word in German and not in English ;)

Reply:
True. It just makes things a bit easier^^